test/Libraries/RevitIntegrationTests/SolidTests.cs

```diff
@@ -4,96 +4,118 @@
 
+using System.IO;
+
+using System.Linq;
+using Autodesk.DesignScript.Geometry;
+
 namespace RevitSystemTests
 {
+    [TestFixture]
     class SolidTests : RevitSystemTestBase
     {
         [Test]
         [TestModel(@".\empty.rfa")]
         public void BlendSolid()
         {
-            //var model = ViewModel.Model;
 
-            //string samplePath = Path.Combine(workingDirectory, @".\Solid\BlendSolid.dyn");
-            //string testPath = Path.GetFullPath(samplePath);
+            string samplePath = Path.Combine(workingDirectory, @".\Solid\BlendSolid.dyn");
+            string testPath = Path.GetFullPath(samplePath);
 
-            //model.Open(testPath);
-            //ViewModel.Model.RunExpression();
+            ViewModel.OpenCommand.Execute(testPath);
 
-            //var blendNode = ViewModel.Model.CurrentWorkspace.Nodes.First(x => x is CreateBlendGeometry);
-            //var result = (Solid)VisualizationManager.GetDrawablesFromNode(blendNode).Values.First();
-            //double volumeMin = 3700000.0;
-            //double volumeMax = 3900000.0;
-            //double actualVolume = result.Volume;
-            //Assert.Greater(actualVolume, volumeMin);
-            //Assert.Less(actualVolume, volumeMax);
+            RunCurrentModel();
+
+            AssertNoDummyNodes();
+
+            Assert.AreEqual(26, ViewModel.Model.CurrentWorkspace.Nodes.Count);
+            Assert.AreEqual(53, ViewModel.Model.CurrentWorkspace.Connectors.Count());
+
+            //check Create List
+            var listID = "238f2bff-076c-482f-9420-7c1111ac4b95";
+            AssertPreviewCount(listID, 2);
+            var list = GetFlattenedPreviewValues(listID);
+            Assert.AreEqual(list[0].ToString(), list[1].ToString());
+
+            //check Solid.Byloft
+            var solidID = "b8d5c31d-e23d-4e18-bb17-ffce4da2f53e";
+            var solid = GetPreviewValue(solidID) as Solid;
+            Assert.IsNotNull(solid);
         }
 
-        [Test]
+
+      /*  [Test]
         [TestModel(@".\empty.rfa")]
         public void Loft()
         {
-            //var model = ViewModel.Model;
+            string samplePath = Path.Combine(workingDirectory, @".\Solid\Loft.dyn");
+            string testPath = Path.GetFullPath(samplePath);
 
-            //string samplePath = Path.Combine(workingDirectory, @".\Solid\Loft.dyn");
-            //string testPath = Path.GetFullPath(samplePath);
+            ViewModel.OpenCommand.Execute(testPath);
 
-            //model.Open(testPath);
-            //ViewModel.Model.RunExpression();
+            RunCurrentModel();
+
+            AssertNoDummyNodes();
 
-            //var fec = new FilteredElementCollector( DocumentManager.Instance.CurrentDBDocument );
-            //fec.OfClass(typeof(GenericForm));
+            Assert.AreEqual(30, ViewModel.Model.CurrentWorkspace.Nodes.Count);
+            Assert.AreEqual(53, ViewModel.Model.CurrentWorkspace.Connectors.Count());
 
-            ////verify one loft created
```

Comment:
@riteshchandawar , Loft.dyn contains warning, and it cannot be tested now. so this method has been commentted